### ISSUE TITLE
Daggers (┼) in taxon names

### ISSUE THREAD

**Issue by aoern** [CatalogueOfLife/data#417]
There are 196 extinct genus and family names with a trailing dagger sign (┼) (in source dataset FLOW, Psyllist and Tessaratomidae Database).
To my understanding, the dagger is not a part of the taxon name, it is only conventionally and optionally used when displaying the name. Now, applications that rely on the extinct flag of the taxon are in trouble and may display double daggers. See the taxon page of Acixiites costalis, for example. It seems to be very very extinct: "†Acixiites† costalis"!
@mdoering, could the taxon name parser just silently remove the dagger?

**Comment by mdoering** [comment]
Yes, the parser does remove the dagger symbol already and instead offer an extinct property. I will see what is going on in these cases, it might be we only expect the dagger in front, not at the end: https://www.catalogueoflife.org/data/taxon/64K7Y

**Comment by mdoering** [comment]
works fine for the species, but still not for the higher uninomials:
https://www.dev.checklistbank.org/dataset/131888/taxon/1425

**Comment by mdoering** [comment]
Now also fixed for denormalised higher taxa: https://www.dev.checklistbank.org/dataset/131888/taxon/9560

**Comment by mdoering** [comment]
@yroskov @gdower you need to update all those sources that have the dagger problem. I only tested it with FLOW.
It needs a new import and then sync.

FLOW btw is a [proxy resource](https://github.com/CatalogueOfLife/data/blob/master/registry/1011.yaml) with data files taken from the Paris server, apparently updated just some weeks ago:
```
curl -s -v -X HEAD "http://www.hemiptera-databases.org/4d4life/flow/AcceptedSpecies.csv"
< Last-Modified: Thu, 31 Mar 2022 22:07:15 GMT
```

The new species [Anotia firebugia Bahder & Bartlett, 2020](https://flow.hemiptera-databases.org/flow/?page=explorer&db=flow&lang=en&card=taxon&rank=species&id=23539) is not in that list though. @ThierryBourgoin it seems there is a problem exporting those "4d4life" files?
